VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/linked-participant-panel-model.ts

```diff
@@ -23,6 +23,10 @@ export class LinkedParticipantPanelModel extends PanelModel {
         return this.participants.some(p => p.isWitness);
     }
 
+    private get participantsInHearing(): PanelModel[] {
+        return this.participants.filter(p => p.isInHearing());
```

Comment:
Looks like the condition is: there are no participants in the hearing with their mic _not_ muted - you could do this
```ts
this.participants.some(p => p.isInHearing() && !p.isLocalMicMuted())
```